src/webviews/api/extension-server/trpc.ts

```diff
@@ -13,7 +13,10 @@
  * @see https://trpc.io/docs/v11/procedures
  */
 
+import { callWithTelemetryAndErrorHandling } from '@microsoft/vscode-azext-utils';
 import { initTRPC } from '@trpc/server';
+// eslint-disable-next-line import/no-internal-modules
+import { type MiddlewareResult } from '@trpc/server/dist/unstable-core-do-not-import/middleware';
```

Comment:
:) Are you sure that this is legal usage? Even folder name says that you have not to import this

Reply:
:) it does look scary but that's just to warn us not to use it as the API might/will change. But it's while they're transiting between versions. Once the new API is there, I'll migrate. It still made sense to work with their newer APIs as the upgrade path will be easier.

Reply:
Please add todo.